ros2_control_demo_description/rrbot_description/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(rrbot_description)
 find_package(ament_cmake REQUIRED)
```

Comment:
I guess this demo doesn't belong in `rrbot_description` because it uses a UR robot. Prob should make a new package & directory for it.

Reply:
done